test/fixtures.yml

```diff
@@ -1451,3 +1451,8 @@ worldpay_us:
   acctid: MPNAB
   subid: SPREE
   merchantpin: "1234567890"
+
```

Comment:
You need to add the entry for your fixtures in fixtures_test.rb. This test is currently failing:

/home/runner/work/active_merchant/active_merchant/test/unit/fixtures_test.rb:6:in `test_sort'

Reply:
Fixed! Now it’s working 😊